sonar-php-plugin/src/main/java/org/sonar/plugins/php/phpunit/PhpUnitResultParser.java

```diff
@@ -238,7 +222,8 @@ private void saveTestsDetails(PhpUnitTestReport fileReport) {
     details.append("</tests-details>");
     InputFile unitTestFile = getUnitTestInputFile(fileReport);
     if (unitTestFile != null) {
-      context.saveMeasure(unitTestFile, new Measure(CoreMetrics.TEST_DATA, details.toString()));
+      context.<String>newMeasure().on(unitTestFile).withValue(details.toString()).forMetric(CoreMetrics.TEST_DATA).save();
```

Comment:
@vilchik-elena agreed. TEST_DATA has been deprecated for MutableTestPlan which im turn is now deprecated